examples/lib/defaultHtmlStepUi.js

```diff
@@ -216,11 +217,16 @@ function DefaultHtmlStepUi(_sequencer, options) {
   function onDraw(step) {
     $(step.ui.querySelector(".load")).show();
```

Comment:
Yes that seems good. Also I was thinking something. `step.ui.querySelector` is used a lot of times in this code so instead maybe we can write a shorthand function which returns `step.ui.querySelector` wrapped in jquery.

Maybe
```js
function stepSelector(queryString){
  return $(step.ui.querySelector(queryString));
}
```

If this idea is good then it can be a new issue even. Maybe an fto-candidate.

Reply:
Definitely @HarshKhandeparkar! Thinking about it now makes me wonder that there must be many more snippets such as this one in the codebase that can be optimized. We can actually open up and pin an issue with a label like `first-timer-opportunity`, promoting among new contributors the practice to find and execute such refactor opportunities hidden in the codebase to make it more readable and submit those refactored codes as `fto` PRs which can help us immensely in the future!! Thoughts?

Reply:
Thats an amazing idea. Maybe we can even add an fto issue to create another issue with many such refactor opportunities and then mark that issue as `break-me-up` and that issue can then be broken up into many more `fto`s. We can also create one issue about this `stepSelector` now and give it as an example. Thoughts?

Reply:
> One thing that I'd like to propose here is storing all these "query selected elements" into their respective _descriptively named_ variables and using them instead so as to make the code more readable. Thanks!

I appreciate your idea, and think that refactoring can be done for the entire project as you guys are discussing. And I think let's keep refactoring separate from this pr. Also if there are any more suggestions , please provide inputs.

Reply:
@jywarren We'd love to hear from you regarding the above ideas. Thanks!